submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/submission/web/resources/SubmissionDraftResource.kt

```diff
@@ -82,28 +85,28 @@ internal class SubmissionDraftResource(
         @RequestBody content: String,
     ): ResponseSubmissionDraft = requestDraftService.createRequestDraft(content, user.email).asResponseDraft()
 
-    @PostMapping("/{key}/submit")
+    @PostMapping("/{accNo}/submit")
     suspend fun submitDraft(
-        @PathVariable key: String,
+        @PathVariable accNo: String,
         @BioUser user: SecurityUser,
         onBehalfRequest: OnBehalfParameters?,
         @ModelAttribute parameters: SubmitParameters,
     ) {
         val buildRequest = SubmitBuilderRequest(user, onBehalfRequest, parameters)
-        val request = submitRequestBuilder.buildDraftRequest(key, user.email, buildRequest)
+        val request = submitRequestBuilder.buildDraftRequest(accNo, user.email, buildRequest)
 
         submitWebHandler.submitAsync(request)
     }
 
-    @PostMapping("/{key}/submit/sync")
+    @PostMapping("/{accNo}/submit/sync")
```

Comment:
at the end how the accNo will be generated for a new draft?

Reply:
It hasn't changed, when it's a new draft the accNo will be `TMP_{Date}`. I'm just getting rid of the `key` word since that concept no longer applies.